lib/editor/components/pattern/CalculateDefaultTimesForm.js

```diff
@@ -95,6 +95,9 @@ export default class CalculateDefaultTimesForm extends Component<Props, State> {
     this.setState({speed: +evt.target.value})
 
   render () {
+    const { activePattern } = this.props
+    const activePatternHasLocations = activePattern.patternLocations.length > 0
```

Comment:
We should check for location groups too here, because those would require the same new logic to actually implement `Normalize Stop Times`

Reply:
Very good point thank you!!!